target/linux/ath79/dts/qca9531_yuncore_cpe830.dts

```diff
@@ -0,0 +1,122 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "yuncore,cpe830", "qca,qca9531";
+	model = "YunCore/KuWfi CPE830(D)";
```

Comment:
YunCore CPE830D (use model name of base device)

Reply:
no, this is incorrect

Your harasmment will lead me to call the police at some point